pkgs/development/python-modules/puremagic/default.nix

```
@@ -0,0 +1,33 @@
+{ pkgs, lib, fetchFromGitHub }:
+
+with pkgs.python3Packages;
```

```suggestion
{ lib, fetchFromGitHub }:
```

Please inputs python packages directly and add ``disabled = isPy27;``.